inc/wpml-compatibility-test-tools.class.php

```diff
@@ -318,6 +322,10 @@ public function register_administration_page() {
 			$this,
 			'load_template'
 		) );
+		add_submenu_page( 'mt', __( 'Custom Fields Translation', 'wpml-compatibility-test-tools' ), __( 'Custom Fields Translation', 'wpml-compatibility-test-tools' ), 'manage_options', 'cf-translations', array(

```

Comment:
@amElnagdy The title is a bit confusing. It makes me think its going to help translate but its really helping me with translation preferences.

Maybe 'Custom Field Settings Helper' would be better? or something along those lines ...